packages/tsort/tsort.1.0.0/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+synopsis: "Easy to use and user-friendly topological sort"
+description: """
+Easy to use and user-friendly topological sort.
+
+Example:
+```
+Tsort.sort [("foundation", []); ("walls", ["foundation"]); ("roof", ["walls"])]
+```
+"""
+maintainer: "daniil@baturin.org"
+authors: ["Daniil Baturin <daniil@baturin.org>"]
+homepage: "https://github.com/dmbaturin/ocaml-tsort"
+bug-reports: "https://github.com/dmbaturin/ocaml-tsort/issues"
+dev-repo: "git+https://github.com/dmbaturin/ocaml-tsort.git"
+license: "MIT"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "containers"
```

Comment:
I would be very surprised if any versions of containers works here. If you could check what is the minimal version required for this dependency that'd be perfect. Otherwise I understand that might be a bit annoying to do and I'm willing to pass on this. It's not mandatory for this PR to be merged.

Reply:
The dependencies of containers 2.7 say 4.02. I've just checked it it works with 4.03 and it actually does.
Since tsort now uses custom a tri-state type for its results to indicate cycles and missing dependencies explicitly, it may work with even earlier versions, but I doubt anyone will want to try 4.02.